### PR TITLE
BDSP First OD Repository - updated

### PR DESCRIPTION
**Pull request by v-mourajr**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/bdsp-sparcnet.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Name: SPaRCNet data:Seizures, Rhythmic and Periodic Patterns in ICU Electroencephalography
+Description: The IIIC dataset includes  50,697 labeled EEG samples from 2,711 patients' and 6,095 EEGs that were annotated by physician experts from 18 institutions. These samples were used to train SPaRCNet (Seizures, Periodic and Rhythmic Continuum patterns Deep Neural Network), a computer program that classifies IIIC events with an accuracy matching clinical experts. 
+Documentation: More documentation can be found [here](https://doi.org/10.60508/cw6j-s785)
+Contact: admin@bdsp.io
+ManagedBy: "[Brain Data Science Platform](https://bdsp.io/)"
+UpdateFrequency: New data is added as soon as it is available.
+Tags:
```
Comment (chueatwork) [inline datasets/bdsp-sparcnet.yaml:7]: @v-mourajr can you add `aws-pds` tag to reflect the sponsorship by the AWS Open Data Program?

File: datasets/bdsp-sparcnet.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - neurophysiology
+  - medicine
+  - machine learning
+  - neuroscience
+  - deep learning
+  - life sciences
+  - bioinformatics
+License: BDSP Restricted Health Data License 1.0.0 "[BDSP Licence](https://bdsp.io/content/bdsp-sparcnet/view-license/1.1/); [Data User Agreement:](https://bdsp.io/content/bdsp-sparcnet/view-dua/1.1/)"
+Resources:
+  - Description: ICU EEG Dataset
```
Comment (chueatwork) [inline datasets/bdsp-sparcnet.yaml:17]: @v-mourajr is this a public access point?

**Comment by v-mourajr** [comment]
Hi Erin, 

Is there anything else that needs to be done to publish the datasets? I submitted a new one today.

Thanks,

Junior

**Comment by chueatwork** [comment]
@v-mourajr please rebase your branch and I can merge!

**Pull request merged**